test/addons/rook-operator/kustomization.yaml

```diff
@@ -42,3 +32,14 @@ patches:
                 env:
                   - name: ROOK_DISABLE_ADMISSION_CONTROLLER
                     value: 'true'
+  - target:
+      kind: ConfigMap
+      name: rook-ceph-operator-config
+      namespace: rook-ceph
+    patch: |-
+      - op: add
+        path: /data/CSI_ENABLE_CSIADDONS
+        value: 'true'
+      - op: add
+        path: /data/CSI_ENABLE_OMAP_GENERATOR
+        value: 'true'
```

Comment:
Any reason you changed he order? Previously we applied this change first, then the other changes. It is a good idea to keep the order the same, unless you have a good reason to change it.